src/senaite/core/browser/controlpanel/instrumentlocations.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+
+import collections
+
+from bika.lims import api
+from bika.lims import bikaMessageFactory as _
+from bika.lims.permissions import AddInstrumentLocation
+from bika.lims.utils import get_link
+from senaite.app.listing import ListingView
+
+
+class InstrumentLocationsView(ListingView):
+    """Displays all available instrument locations in a table
+    """
+
+    def __init__(self, context, request):
+        super(InstrumentLocationsView, self).__init__(context, request)
+
+        self.contentFilter = {
+            "portal_type": "InstrumentLocation",
+            "sort_on": "sortable_title",
+        }
+
+        self.context_actions = {
+            _("Add"): {
+                "url": "++add++InstrumentLocation",
+                "permission": AddInstrumentLocation,
+                "icon": "++resource++bika.lims.images/add.png",
+            }}
+
+        t = self.context.translate
+        self.title = t(_("Instrument Locations"))
+        self.description = t(_(
+            "The place where the instrument is located in the laboratory"))
+
+        self.show_select_column = True
+        self.pagesize = 25
```

Comment:
I wouldn't set the pagesize explicitly. I think is 50 by default, which is fine imo

Reply:
The pagesize is set to 25 for all of the controlpanel listings and I would like to keep them consistent for now.
Maybe we can make a setup option and refactor all of them in another PR?

Reply:
Sounds a good idea. Let's do it in another PR then